### ISSUE TITLE
null value should be displayed appropriately

### ISSUE THREAD

**Issue by hongsudt** [informatics-isi-edu/chaise#519]
By default, I think it should be an empty string.

**Comment by karlcz** [comment]
See also 2015:display `show_nulls` instructions.

**Comment by hongsudt** [comment]
@howdyjessie this should be handled as part of the pretty print functionality.

**Comment by hongsudt** [comment]
This issue is dependent on informatics-isi-edu/ermrestjs#152. 

The app then should check the Tuple.values. If the column value is null, then that column should not be printed, otherwise, print whatever is in that value. The ermrestjs will deal with setting the right null display for the context.

**Comment by howdyjessie** [comment]
This issue is currently blocked/paused due to discrepancies in how we specify contexts for annotations (i.e. `"record"` and `"detailed"` both refer to entity-level apps).

Currently the `show_nulls` property in the `2015:display` annotation uses the `detailed` context to determine a column's `formatvalue` when that column has a `null` value. In Record 2, however, the app uses `Reference.contextualize.record`, which sets the app's context to `"record"`, not `"detailed"`. Because of this, ermrestjs is unable to return the correct tuple value for null columns. 

@RFSH and I consulted with @robes about this, and we've decided the first step to resolving this discrepancy is by editing the [annotations doc](https://github.com/informatics-isi-edu/ermrest/blob/master/user-doc/annotation.md) to codify the contexts we'd like to use consistently. @RFSH will make a PR with changes, and we can discuss what terms to use in this PR thread.
